### PR TITLE
Migrate to new python template

### PR DESCRIPTION
**Pull request by em-pe**

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -5,20 +5,11 @@ repos:
...
   - id: black
 - repo: https://github.com/pre-commit/pre-commit-hooks
   rev: v2.3.0
   hooks:
   - id: flake8
-    args: ['--ignore=E203,W503'] # see https://github.com/psf/black/issues/315 https://github.com/psf/black/issues/52
-- repo: https://github.com/getindata/py-pre-commit-hooks
-  rev: v0.1.3
-  hooks:
-  - id: pyspelling-docker
```
Comment (szczeles) [inline .pre-commit-config.yaml:19]: Why do we get rid of both hooks here?

File: pyproject.toml [hunk 2]
```diff
@@ -1,2 +1,63 @@
-[tool.black]
-line-length = 79
```
Comment (szczeles) [inline pyproject.toml:2]: Let's keep the original setting for `black`, so the code should remain untouched

**Review by szczeles — APPROVED** [review]
Looks good! Just 2 minors left

File: LICENSE [hunk 3]
```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2022 GetInData
```
Comment (szczeles) [inline LICENSE:189]: Actually these brackets make sense if you look at the section above ;-)

File: .pre-commit-config.yaml (hunk 1 above)
Comment (em-pe) [reply .pre-commit-config.yaml]: good catch, I'll add those to the template as well.

File: pyproject.toml (hunk 2 above)
Comment (em-pe) [reply pyproject.toml:2]: done

File: LICENSE (hunk 3 above)
Comment (em-pe) [reply LICENSE]: right :)

**Pull request merged**